Bottom: c34d596a36745b354344ac0db316f63fd60390a7
Top:    906cd6662b83b0691c6489b81cbe0f40311d9e1b
Author: Marina Samuel <msamuel@mozilla.com>
Date:   2018-11-27 10:18:34 -0500

Use entry_point.module_name instead of entry_point.name for looking up the bundles.


---
diff --git a/bin/bundle-extensions b/bin/bundle-extensions
index 8416aab7..56173f4c 100755
--- a/bin/bundle-extensions
+++ b/bin/bundle-extensions
@@ -22,9 +22,12 @@ os.environ["EXTENSIONS_DIRECTORY"] = EXTENSIONS_RELATIVE_PATH
 for entry_point in iter_entry_points('redash.extensions'):
     # This is where the frontend code for an extension lives
     # inside of its package.
-    content_folder_relative = os.path.join(
-        entry_point.name, 'bundle')
-    (root_module, _) = os.path.splitext(entry_point.module_name)
+
+    split_module_path = entry_point.module_name.split(os.extsep)
+    root_module = split_module_path.pop(0)
+
+    content_folder_relative = os.path.join(os.path.join(
+        *split_module_path), 'bundle')
 
     if not resource_isdir(root_module, content_folder_relative):
         continue
diff --git a/requirements.txt b/requirements.txt
index d2d30639..4269850a 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -61,4 +61,4 @@ disposable-email-domains
 # It is not included by default because of the GPL license conflict.
 # ldap3==2.2.4
 gevent==1.4.0
-redash-stmo>=2018.9.1
+redash-stmo>=2018.12.0
